website/docs/r/tm_content_library.html.markdown

```diff
@@ -37,7 +37,7 @@ resource "vcd_tm_content_library" "cl" {
 The following arguments are supported:
 
 * `name` - (Required) The name of the Content Library
-* `storage_policy_ids` - (Required) A set of [Region Storage Policy](/providers/vmware/vcd/latest/docs/resources/tm_region_storage_policy) IDs or VDC Storage Policy IDs used by this Content Library
+* `storage_class_ids` - (Required) A set of [Storage Class IDs](/providers/vmware/vcd/latest/docs/data-sources/tm_region_storage_policy) used by this Content Library
```

Comment:
If it's renamed to "class", why are we still using "policy" in data source?

Reply:
The name in its API didn't change for Region Storage Policies (yet?). I guess it will be done "soon" to have both aligned and we will be able to change the data source name then

Reply:
Let's add a TODO: TM: here as a reminder to check for API change.

Reply:
Done